src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ProjectTypeRegistration.cs

```diff
@@ -45,6 +45,7 @@
     Capabilities = ProjectTypeCapabilities.CSharp,
     DisableAsynchronousProjectTreeLoad = true,
     PossibleProjectExtensions = "csproj",
-    NewProjectRequireNewFolderVsTemplate = true)]
+    NewProjectRequireNewFolderVsTemplate = true,
+    SupportsCodespaces = true)]
```

Comment:
I thought this was for all .NET core projects. Is there a reason VB and F# are not included currently?

Reply:
Agreed; both C# and VB need to be opt'd in.

How does this interplay with selectors? The GUID persisted in the solution is the legacy project system, which the selector redirects to this one. Does this new property get handled by that?  @davidwengier

Reply:
What about Shared Projects?

Reply:
> How does this interplay with selectors? The GUID persisted in the solution is the legacy project system, which the selector redirects to this one. Does this new property get handled by that? @davidwengier

We discussed this, and I don't know. It needs to be tested. It depends exactly where the code is that reads this (or perhaps should I say "when" the code is) as to whether its after the pre-fetch or not.

Reply:
> What about Shared Projects? That registration is in the VS repository.

t's in the CPS repo - codesharingproj.pkgdef. And yes, that should also be opted in. @arkalyanms

Reply:
> Agreed; both C# and VB need to be opt'd in.
@davkean Made the change assuming you meant F# and VB.

Shared projects PR - https://devdiv.visualstudio.com/DevDiv/_git/CPS/pullrequest/250253